Calibration/TkAlCaRecoProducers/python/ALCARECOPromptCalibProd_cff.py

```diff
@@ -83,6 +90,7 @@
                                           ALCARECOHltFilterForBS *
 #                                           offlineBeamSpotForBS +
 #                                           offlinePrimaryVerticesForBS +
-                                          alcaBeamSpotProducer)
+                                          alcaBeamSpotProducer +
+                                          alcaBeamSpotProducerHP)
```

Comment:
as agreed verbally with  @rmanzoni 
we defer the usage of this PR for the next iteration